src/segments/python.go

```diff
@@ -72,13 +74,23 @@ func (p *Python) loadContext() {
 		"CONDA_ENV_PATH",
 		"CONDA_DEFAULT_ENV",
 	}
+	defaultVenvNames := []string{
```

Comment:
can you add a space between this new block? Above and below, will improve readability